### PR TITLE
Update generate_fasta.py to handle inframe indels and frameshift mutations

### PR DESCRIPTION
**Pull request by susannasiebert**

### CODE REVIEW

**Comment by chrisamiller** [comment]
+1

**Comment by agraubert** [comment]
+1

**Comment by acoffman** [comment]
+1

File: pvacseq/lib/generate_fasta.py [hunk 1]
```diff
@@ -11,70 +14,134 @@ def distance_from_start(position, string):
...
+        consequence_string = line['consequences']
+        if consequence_string is None:
+              continue
+        consequences = {consequence.lower() for consequence in consequence_string.split('&')}
+
+        full_wildtype_sequence = line['wildtype_amino_acid_sequence']
+        if 'frameshift_variant' in consequences:
+            consequence = 'FS'
+            position = int(line['protein_position']) - 1
+        elif 'missense_variant' in consequences or 'inframe_insertion' in consequences:
```
Comment (acoffman) [inline pvacseq/lib/generate_fasta.py]: Checking this here, and then immediately re-checking it on the lines below seems less than ideal. Perhaps you could store the result somehow? But I can't think of a super elegant way to handle it offhand.

**Pull request merged**